Tasks/MSBuild/Select-MSBuildLocation.ps1

```diff
@@ -32,7 +32,11 @@ function Select-MSBuildLocation {
 
             # Look for a specific version of MSBuild.
             if ($Version -and "$Version".ToUpperInvariant() -ne 'LATEST') {
-                $Location = Get-MSBuildPath -Version $Version -Architecture $Architecture
+
+                # Only search Willow installs if 15.0 was explicitly chosen.
+                $searchCom = $Version -eq "15.0"
```

Comment:
The COM interface is only searched when "15.0" is selected. If the user selects "Latest", the COM interface is not searched. This is a targeted fix for RC2 only. Before RTM, we should remove all this "search COM" branching logic and just unconditionally attempt to query the COM interface if it's registered.